packages/typescript-estree/src/parser.ts

```diff
@@ -14,8 +14,8 @@ import ts from 'typescript';
 import convert from './ast-converter';
 import { convertError } from './convert';
 import { firstDefined } from './node-utils';
-import * as es from './typedefs';
-import { Extra, ParserOptions } from './parser-options';
+import * as TSESTree from './typedefs';
```

Comment:
I was going to suggest this change (es -> TSESTree) in your last PR but didn't want to hold it up over something so trivial.

One thing though, can you go back and make this usage consistent? We use `es.` in other places than the ones which have been updated here.

Maybe do a barrel style export of a named symbol for all, and then import from the barrel instead?

```ts
// barrel
import * as TSEStree from '...'
export { TSEStree }
```

Then refactoring might be easier in future?

Reply:
i renames all es to TSEStree, i'm unsure what editor you are using but in Intelij its 
`right click` -> `refactor` -> `rename` -> and its done xd

Reply:
Left a comment below